### DISCUSSION TITLE
Trichrome library

### DISCUSSION THREAD

**Discussion by CrisMystik** [bromite/bromite#1358] [Q&A] [answered]
I know that on Android 10+, Chrome and WebView require the Trichrome library.
Does Bromite (and Bromite SystemWebView) need it in the same way?

**Comment by ConradGyllenhaal** [comment, marked as answer]
If you take Bromite from F-Droid or GitHub releases section, it can be a standalone app, no needing the Trichrome Library.

If you want to build Bromite by yourself, you can do it the "Trichrome" way as it is done in the [CHAOSP projetct](https://github.com/CaseyBakey/chaosp) for example.

**Comment by csagan5** [comment]
As far as I remember the Trichrome library is needed only if you bundle the SystemWebView in the ROM, to take advantage of an optimization (shared library/resources with the browser APK).

So it is not needed but it could be an advantage in that situation.
